app/controllers/conversations.js

```diff
@@ -144,9 +210,35 @@ class ConversationsController extends BaseController {
               conversation_id: conversationId,
             }))
           ) {
-            await obj.save();
+            await participant.save();
+            participants.push(participant.params._id);
           }
         }
+
+        if (participants.length) {
+          const currentUserLogin = (await User.findOne({ _id: currentUserId }))
+            ?.params?.login;
+          await PacketProcessor.deliverToUserOrUsers(
```

Comment:
same here, let's move it to a separate private method in this controller and do not duplicate it 3 times